server/core/src/miscServerFunct.cpp

```diff
@@ -65,6 +71,7 @@ char *__loc1;
 #include <boost/filesystem.hpp>
 
 using leaf_bundle_t = irods::resource_manager::leaf_bundle_t;
+namespace bp = boost::process;
```

Comment:
Given that Boost.Process is used in only one function, we can move this into the function.